src/olm/account/mod.rs

```diff
@@ -129,8 +129,8 @@ impl Account {
     }
 
     /// Sign the given message using our Ed25519 fingerprint key.
-    pub fn sign(&self, message: &str) -> Ed25519Signature {
-        self.signing_key.sign(message.as_bytes())
+    pub fn sign(&self, message: &[u8]) -> Ed25519Signature {
```

Comment:
This could take an `impl AsRef<[u8]>` to avoid a breaking change and to streamline it with `Sesssion::encrypt()`.